WeakAuras/RegionTypes/Icon.lua

```diff
@@ -599,6 +614,14 @@ local function modify(parent, region, data)
     end
   end
 
+  function region:SetCooldownSwipe(drawswipe)
```

Comment:
```suggestion
  function region:SetCooldownSwipe(cooldownSwipe)
    region.cooldownSwipe = cooldownSwipe
```

Reply:
why do you want to set region.cooldownSwipe here? region:SetInverse is analogous to this function but that doesn't change its corresponding value either.

Reply:
For consistency's sake, I would like to move towards all of the region methods setting the values they get to the region (and also maybe validating them). I understand that SetInverse doesn't do this, but it's the direction I want to take this. Note the glow methods, color, coloranim, scale, width & height, etc. Not every method does this appropriately, but I believe that it should.

Reply:
Those functions only do that, because they check if the values changed and do a early return based on that. If there's no early return I don't see a particular value in having the values in region.